t/if-then-else.c

```diff
@@ -45,9 +45,9 @@ static void out(void *ctx, const char *p, size_t n) {
 }
 
 int main(void) {
-    unibi_var_t var_dyn   [26] = {0};
-    unibi_var_t var_static[26] = {0};
-    unibi_var_t param[9] = {0};
+    unibi_var_t var_dyn   [26] = {{0}};
```

Comment:
the warning about missing braces in the universal initializer is bogus and was removed in recent gcc versions
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53119